src/libraries/System.Private.CoreLib/src/System/Reflection/NullabilityInfoContext.cs

```diff
@@ -71,13 +72,11 @@ public NullabilityInfo Create(ParameterInfo parameterInfo)
 
             EnsureIsSupported();
 
-            if (parameterInfo.Member is MethodInfo method && IsPrivateOrInternalMethodAndAnnotationDisabled(method))
-            {
-                return new NullabilityInfo(parameterInfo.ParameterType, NullabilityState.Unknown, NullabilityState.Unknown, null, Array.Empty<NullabilityInfo>());
-            }
-
             IList<CustomAttributeData> attributes = parameterInfo.GetCustomAttributesData();
-            NullabilityInfo nullability = GetNullabilityInfo(parameterInfo.Member, parameterInfo.ParameterType, attributes);
+            NullableAttributeStateParser parser = parameterInfo.Member is MethodBase method && IsPrivateOrInternalMethodAndAnnotationDisabled(method)
+                ? NullableAttributeStateParser.Unknown
+                : CreateParser(attributes);
```

Comment:
This change (and similar) allows us to still capture nullability info from value types and code analysis attributes even when the compiler-generated nullable annotations are missing.